apps/liquidity-provision-dashboard/src/app/components/header/header.tsx

```diff
@@ -0,0 +1,13 @@
+import { t } from '@vegaprotocol/react-helpers';
+const Header = () => {
+  return (
+    <div className="flex items-stretch px-6 py-6" data-testid="header">
+      <h1 className="text-3xl">{t('Top liquidity opportunities')}</h1>
+      <div className="flex items-center gap-2 ml-auto relative z-10">
+        {t('Network switcher')}
+      </div>
+    </div>
+  );
+};
+
+export default Header;
```

Comment:
chore (blocking): Please use named exports and avoid default exports unless default is 100% required (usually only for lazy loading of components)